src/namecoin.py

```diff
@@ -159,19 +158,19 @@ def test(self):
                     versStr = "0.%d.%d.%d" % (v1, v2, v3)
                 message = (
                     'success',
-                    tr._translate(
+                    _translate(
                         "MainWindow",
                         'Success!  Namecoind version %1 running.').arg(
-                            unicode(versStr)))
+                            versStr.decode('utf-8', 'ignore')))
 
             elif self.nmctype == "nmcontrol":
                 res = self.callRPC("data", ["status"])
                 prefix = "Plugin data running"
                 if ("reply" in res) and res["reply"][:len(prefix)] == prefix:
-                    return ('success', tr._translate("MainWindow", 'Success!  NMControll is up and running.'))
+                    return ('success', _translate("MainWindow", 'Success!  NMControll is up and running.'))
```

Comment:
line too long

Reply:
Also it's better to use double quotes in `_translate()` calls for uniformity:
```python
return (
    'success',
    _translate("MainWindow", "Success!  NMControll is up and running."))
```

Reply:
updated